ansible/docker/Dockerfile.CentOS7

```diff
@@ -3,9 +3,21 @@ FROM centos:7
 ARG git_sha
 ARG user=jenkins
 
-RUN sed -i -e 's!mirrorlist!#mirrorlist!g' /etc/yum.repos.d/CentOS-Base.repo; \
-    sed -i -e 's!#baseurl=http://mirror.centos.org/centos/\$releasever!baseurl=https://vault.centos.org/7.9.2009/!g' /etc/yum.repos.d/CentOS-Base.repo; \
-    yum -y update; yum clean all; \
+RUN sed -i -e 's!mirrorlist!#mirrorlist!g' /etc/yum.repos.d/CentOS-Base.repo
+
+RUN if [ "$(uname -m)" = "aarch64" ]; then \
```

Comment:
Maybe I'm missing something but given that the non-x64 ones are the same apart from the echo statement, is there any reason not to collapse them and use `if [ "$(uname -m)" != "aarch64" ];` to keep this simpler?

Reply:
Good point, let me tidy that... I was expecting them to be different when I wrote the if statement, whilst I figured out the sed goodness...

Reply:
Fixed